packages/bpk-stylesheets/index.scss

```diff
@@ -16,65 +16,72 @@
  * limitations under the License.
  */
 
-@import '~normalize.css';
-@import '../bpk-mixins/index.scss';
+@layer reset, base, component, utility;
```

Comment:
To prevent naming clashes in consumers who also use named layers could these all have the `bpk-` prefix? The latter 3 are highly likely to be names a consumer is/wishes to use.

I know this is WIP, just commenting ahead of names potentially becoming baked in and harder to swap out.

Reply:
We've recently encountered race-condition bugs here related to the repeated use of the name 'component' so yes I would agree that something like this would enable further protection in case the consumer uses these names. We did discuss that as an option ages ago but it didn't seem very consequential at the time.

Reply:
Thanks for the comment @dominicfraser let us take this away and chat about it.

> The latter 3 are highly likely to be names a consumer is/wishes to use.

This could also be influenced by the guidance we provide too of possible avoid these, as we will want to make sure whatever we use/provide to folks is consistent across the product too instead of just leaving it to consumers to come up with namings

Reply:
Have added 'bpk-' prefix